pkg/oc/cli/admin/cert/cert.go

```diff
@@ -1,33 +1,32 @@
 package cert
 
 import (
-	"io"
-
 	"github.com/spf13/cobra"
 	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	"k8s.io/kubernetes/pkg/kubectl/genericclioptions"
 
 	"github.com/openshift/origin/pkg/cmd/server/admin"
 )
 
 const CertRecommendedName = "ca"
 
 // NewCmdCert implements the OpenShift cli ca command
-func NewCmdCert(name, fullName string, out io.Writer, errout io.Writer) *cobra.Command {
+func NewCmdCert(name, fullName string, streams genericclioptions.IOStreams) *cobra.Command {
 	// Parent command to which all subcommands are added.
 	cmds := &cobra.Command{
 		Use:   name,
 		Short: "Manage certificates and keys",
 		Long:  `Manage certificates and keys`,
-		Run:   cmdutil.DefaultSubCommandRun(errout),
+		Run:   cmdutil.DefaultSubCommandRun(streams.ErrOut),
```

Comment:
Maybe a bit out of scope with these particular changes, but could you create an options struct for this command to hold the iostreams?